### ISSUE TITLE
Native HLS Subtitle Support

### ISSUE THREAD

**Issue by samueleastdev** [silvermine/videojs-chromecast#71]
Hi All,

Great plugin just wondering if this plugin supports Native hls subtitles like in this stream?

https://d2zihajmogu5jn.cloudfront.net/bipbop-advanced/bipbop_16x9_variant.m3u8

Example: https://codepen.io/raestrada/pen/rbZqQG

I see there is an option here.

https://developers.google.com/cast/docs/reference/player/cast.player.api.Player#enableCaptions

But I see in your code it is disabled
~~~
// Text tracks are not supported in this version
   ChromecastTechImpl.prototype.featuresNativeTextTracks = false;
   ChromecastTechImpl.prototype.featuresNativeAudioTracks = false;
   ChromecastTechImpl.prototype.featuresNativeVideoTracks = false; 
~~~

Just wanted to check if I am missing something.

Thanks

**Comment by jthomerson** [comment]
I don't think so. PR's welcome. Please just follow our [coding standards](https://github.com/silvermine/silvermine-info/blob/master/coding-standards.md) (eslint should take care of most of it for you) and [commit guidelines](https://github.com/silvermine/silvermine-info/blob/master/commit-history.md).
